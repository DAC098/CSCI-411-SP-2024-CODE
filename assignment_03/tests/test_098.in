4 10
16 107 387 883
4088
893
8822
6269
8374
9496
7507
1321
3404
5870
