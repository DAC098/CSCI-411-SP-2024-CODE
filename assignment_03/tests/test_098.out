0 0 6 2 
9 7 0 0 
0 4 8 6 
2 2 11 2 
3 4 9 5 
7 1 8 7 
0 2 12 3 
10 0 3 0 
10 5 7 0 
1 0 6 4 
